postoverflows/s01-whitelist/crowdsecurity/seo-bots-whitelist.yaml

```diff
@@ -1,5 +1,6 @@
 name: crowdsecurity/seo-bots-whitelist
 description: "Whitelist good search engine crawlers"
+debug: true
```

Comment:
Is this intended ?

Reply:
oh you are damn right. by the way, isn't something we might want to detect / prevent ?